### PR TITLE
tests: Ignore stage3d_raytrace on WARP

### PR DESCRIPTION
**Pull request by Dinnerbone**
The test seems to be flaky on warp

Also reduce the number of max outliers since the vast majority are just a delta of 1

### CODE REVIEW

File: tests/tests/swfs/avm2/stage3d_raytrace/test.toml [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
+tolerance = 1
 # FIXME - see if we can decrease this. Ideally, it wouldn't be nearly so high
 max_outliers = 5100
 
 [player_options]
 # This test runs very slowly on unoptimized `test` builds,
 # so give it plenty of time.
 max_execution_duration = { secs = 1000, nanos = 0 }
-with_renderer = { optional = false, sample_count = 1 }
+with_renderer = { optional = false, sample_count = 1, exclude_warp = true }
```
Comment (Aaron1011) [inline tests/tests/swfs/avm2/stage3d_raytrace/test.toml:16]: Can you add a link to https://github.com/gfx-rs/wgpu/issues/3193 ?

Comment (Dinnerbone) [reply tests/tests/swfs/avm2/stage3d_raytrace/test.toml:16]: I'll rebase & add that in a comment once I see the test results :D

**Review by Aaron1011 — APPROVED** [review]

**Comment by Dinnerbone** [comment]
Just noticed it's still running the test, hmm 🤔

File: tests/tests/util/options.rs [hunk 2]
```diff
@@ -245,13 +251,25 @@ impl ImageComparison {
...
             sample_count: 1,
+            exclude_warp: false,
         }
     }
 }
+
+impl RenderOptions {
+    pub fn is_supported(&self, adapter: &wgpu::Adapter) -> bool {
+        let info = adapter.get_info();
+        if self.exclude_warp && cfg!(windows) && info.vendor == 5140 && info.device == 140 {
```
Comment (torokati44) [inline tests/tests/util/options.rs:270]: Perhaps an explanation, or a link, explaining where these magic values came from, or what they mean, would be nice here.

**Review by torokati44 — APPROVED** [review]

**Pull request merged**